libraries/Microsoft.Bot.Builder.Expressions/BuiltInFunctions.cs

```diff
@@ -495,6 +495,20 @@ public static EvaluateExpressionDelegate ApplyWithError(Func<IReadOnlyList<dynam
                 return (value, error);
             };
 
+        private static (object value, string error) EvaluateExpression(Expression expression, object state)
```

Comment:
I don't get why we need this function?

Reply:
i don't expect this should change expression library.

Reply:
Reverted changes in expression. Current changes in builtinFunction is handling code style